bundle-workflow/scripts/components/k-NN/build.sh

```diff
@@ -63,6 +63,12 @@ mkdir -p $OUTPUT/libs
 
 # Build knnlib and copy it to libs
 cd jni
+
+# For x64, generalize arch so library is compatible for processors without simd instruction extensions
+if ["$ARCHITECTURE" = "x64"]; then 
```

Comment:
Nit: Makes it ` if [ "$ARCHITECTURE" = "x64" ]; then` for a better spacing?

Reply:
Updated.